easybuild/easyblocks/i/icc.py

```diff
@@ -105,53 +112,81 @@ def sanity_check_step(self):
     def make_module_req_guess(self):
         """Customize paths to check and add in environment.
         """
+        # New Directory Layout for Intel Parallel Studio XE 2016
+        # https://software.intel.com/en-us/articles/new-directory-layout-for-intel-parallel-studio-xe-2016
+        debuggerpath = 'debugger_%s' % self.version.split('.')[0]
         if self.cfg['m32']:
             # 32-bit toolchain
+            libpaths = ['lib', 'lib/ia32'],
             dirmap = {
                 'PATH': ['bin', 'bin/ia32', 'tbb/bin/ia32'],
-                'LD_LIBRARY_PATH': ['lib', 'lib/ia32'],
-                'LIBRARY_PATH': ['lib', 'lib/ia32'],
+                'LD_LIBRARY_PATH': libpaths,
+                'LIBRARY_PATH': libpaths,
                 'MANPATH': ['man', 'share/man', 'man/en_US'],
                 'IDB_HOME': ['bin/intel64']
             }
         else:
-            # 64-bit toolit
-            dirmap = {
-                'PATH': ['bin', 'bin/intel64', 'tbb/bin/emt64'],
-                'LD_LIBRARY_PATH': ['lib', 'lib/intel64'],
-                'LIBRARY_PATH': ['lib', 'lib/intel64'],
-                'MANPATH': ['man', 'share/man', 'man/en_US'],
-                'IDB_HOME': ['bin/intel64']
-            }
+            if LooseVersion(self.version) < LooseVersion("2016"):
+                # 64-bit toolkit
+                libpaths = ['compiler/lib/intel64', 'lib/intel64', 'debugger/ipt/intel64/lib', 'ipp/lib/intel64', 'tbb/lib/intel64']
+                dirmap = {
+                    'PATH': ['bin/intel64', 'tbb/bin/intel64', 'ipp/bin/intel64', 'debugger/gdb/intel64/bin'],
+                    'LD_LIBRARY_PATH': libpaths, 
+                    'LIBRARY_PATH': libpaths,
+                    'MANPATH': ['man', 'share/man', 'man/en_US', 'debugger/gdb/intel64/share/man'],
+                    'CPATH': ['ipp/include', 'tbb/include'],
+                    'INTEL_PYTHONHOME': ['debugger/python/intel64']
+                }
```

Comment:
nope. Intel dropped IDB in favor of GDB.
Though only these two got set:
GDBSERVER_MIC=/glb/home/s_hpcssw/easybuildinstall/software/rhel/6/icc/2016.0.109/debugger_2016/gdb/targets/mic/bin/gdbserver
GDB_CROSS=/glb/home/s_hpcssw/easybuildinstall/software/rhel/6/icc/2016.0.109/debugger_2016/gdb/intel64_mic/bin/gdb-mic